docs/src/app/(public)/(content)/react/overview/releases/page.mdx

```diff
@@ -3,6 +3,244 @@
 <Subtitle>Changelogs for each Base UI release.</Subtitle>
 <Meta name="description" content="Changelogs for each Base UI release." />
 
+## v1.0.0-beta.5
+
+**Nov 13, 2025**
+
+### Accordion
+
+- **Breaking change:** Change `multiple` prop to be false by default and add a demo ([#3141](https://github.com/mui/base-ui/pull/3141))
+- Fix flaky exit transition ([#3101](https://github.com/mui/base-ui/pull/3101))
+
+### Alert Dialog
+
+- Fix `initialFocus` as function being called on close ([#2949](https://github.com/mui/base-ui/pull/2949))
+- Support detached triggers ([#2974](https://github.com/mui/base-ui/pull/2974))
+- Place `overflow: hidden` on `<body>` for overlay scrollbars ([#3083](https://github.com/mui/base-ui/pull/3083))
+
+### All Components
+
+- **Breaking change:** Replace `trackAnchor` with `disableAnchorTracking`.
+
+  If you were using `trackAnchor={false}`, be sure to update your usage to `disableAnchorTracking` instead.
+  ([#3188](https://github.com/mui/base-ui/pull/3188))
+
```

Comment:
I've moved descriptions to a separate line item paragraph to avoid an insane wall of text. 🙈 

<img width="807" height="247" alt="Image" src="https://github.com/user-attachments/assets/3e4f66d0-a4e7-4ba9-9a65-530979e916a7" />